src/services/elasticSearchDomain/connections.ts

```diff
@@ -59,6 +70,91 @@ export default ({
     }
   }
 
+  /**
+   * Find any cognito identity pool related data
+   */
+  const identityPools = data.find(
+    ({ name }) => name === services.cognitoIdentityPool
+  )
+  if (identityPools?.data?.[region]) {
+    const dataAtRegion: RawAwsCognitoIdentityPool[] = identityPools.data[
+      region
+    ].filter(
+      ({ IdentityPoolId: poolId }: RawAwsCognitoIdentityPool) =>
+        poolId === IdentityPoolId
+    )
+    for (const identityPool of dataAtRegion) {
+      connections.push({
+        id: identityPool.IdentityPoolId,
+        resourceType: services.cognitoIdentityPool,
+        relation: 'child',
+        field: 'cognitoIdentityPool',
+      })
+    }
+  }
+
+  /**
+   * Find any cognito user pool related data
+   */
+  const userPools = data.find(({ name }) => name === services.cognitoUserPool)
+  if (userPools?.data?.[region]) {
+    const dataAtRegion: RawAwsCognitoUserPool[] = userPools.data[region].filter(
+      ({ Id }: RawAwsCognitoUserPool) => Id === UserPoolId
+    )
+    for (const userPool of dataAtRegion) {
+      connections.push({
+        id: userPool.Id,
+        resourceType: services.cognitoUserPool,
+        relation: 'child',
+        field: 'cognitoUserPool',
+      })
+    }
+  }
+
+  /**
+   * Find any IAM role related data
+   */
+  const roles = data.find(({ name }) => name === services.iamRole)
+  if (roles?.data?.[globalRegionName]) {
+    const dataAtRegion: RawAwsIamRole[] = roles.data[globalRegionName].filter(
+      ({ Arn }: RawAwsIamRole) => Arn === RoleArn
+    )
+    for (const role of dataAtRegion) {
+      connections.push({
+        id: role.Arn,
+        resourceType: services.iamRole,
+        relation: 'child',
+        field: 'iamRole',
+      })
+    }
+  }
+
+  /**
+   * Find any cloudwatch log group related data
+   */
+  const cloudwatchLogGroups = data.find(
+    ({ name }) => name === services.cloudwatchLog
+  )
+  if (cloudwatchLogGroups?.data?.[region]) {
+    const dataAtRegion: RawAwsLogGroup[] = cloudwatchLogGroups.data[
+      region
+    ].filter(
+      ({ arn }: RawAwsLogGroup) =>
+        !isEmpty(logGroupsArns) &&
+        logGroupsArns.filter(
+          str => `${str}:*`.includes(arn) // A small interpolation hack to be able to match the full arn
```

Comment:
hmm can you provide an example here of what str might be and what the arn might be?

Reply:
I found out that Cloudwatch log groups ARNs have that wildcard attached at the end.

Like this for example

![Screenshot from 2022-04-14 14-33-25 (1)](https://user-images.githubusercontent.com/22185836/163455728-6184c994-b9eb-4d95-ad80-1fd35e2b6fea.png)